src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1395,11 +1395,8 @@ public void testHaltOnExceptionOff() throws Exception {
         checkerConfig.addProperty("haltOnException", "false");
 
         final String filePath = getPath("InputChecker.java");
-        final String[] expected = {
-            "1: " + getCheckMessage(EXCEPTION_MSG, "java.lang.IndexOutOfBoundsException: test"),
-        };
 
-        verify(checkerConfig, filePath, expected);
+        execute(checkerConfig, filePath);
```

Comment:
update Input file with XML config in comment and do verifyWithInlineXmlConfig

Reply:
add this CheckWhichThrowsError to https://github.com/checkstyle/checkstyle/blob/66ba088787cd4fdaac9b4187ef28a046fe971b22/src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java#L61

Reply:
Done